README.md

```diff
@@ -1,288 +1,133 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 World-Information<a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**World-Information**
+world-population is a single-page website where users can view all countries in the world and can see their information e.g their population , language, nativeName etc.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
+    React.js
+    React , Reduc Toolkit
+    Javascript(ES6)
+    Webpack
+    Node.js
+    jest
+    Html/CSS
 
-> Describe between 1-3 key features of the application.
+## Key Features <a name="key-features"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **User can see total continent's countries**
+  **User can view every countinent countries detail's e.g population , language, timezones, region, area, native name**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+-[Live Version](https://world-kx2q.onrender.com/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+- Create a local directory that you want to clone the repository.
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- Open the command prompt in the created directory.
 
-### Install
+- On the terminal run this command git clone ``
 
-Install this project with:
+- Go to the repository folder using command prompt cd `World-info-React-Capstone`
 
-<!--
-Example command:
+- Install the dev dependencies for linters run npm install.
+- setup Webpack
 
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Mandatory

- your app should have unit tests for several components of the app and some pure functions in Redux code.
your app should have integration tests using React Testing Library
   * you should perform a mocking data fetching and Redux Store connection.